Mathlib/RingTheory/PowerSeries/Restricted.lean

```diff
@@ -0,0 +1,227 @@
+import Mathlib.Analysis.Normed.Group.Ultra
+import Mathlib.Analysis.RCLike.Basic
+import Mathlib.Data.Real.StarOrdered
+import Mathlib.RingTheory.PowerSeries.Basic
+import Mathlib.Analysis.Normed.Ring.Ultra
+
+variable (c : NNReal) (R : Type*) [NormedRing R]
+
+open PowerSeries Filter IsUltrametricDist
+open scoped Topology
+
+/-- The convergence property for restricted powerseries. -/
+def Convergent (f : PowerSeries R) : Prop :=
```

Comment:
There also seems to be a more general definition of restricted power series available for arbitrary topological rings - is your plan to connect the two at a later stage? It may be worth discussing this on Zulip to ask for maintainer feedback on this!